### DISCUSSION TITLE
Request to being pinged in Rust PRs and the like

### DISCUSSION THREAD

**Discussion by Techie-Pi** [rust3ds/ctru-rs#79] [General]
I really like this project, and I try to stay up-to-date with things like the threads PR and its ACP. I know the target maintainers are always pinged (for obvious reasons), but I would also like to be pinged on future Rust PRs, issues and things like that if that's alright with everyone.

I want to contribute to this as much as my knowledge and time allows me to, because this is a project that has really helped me to get into the 3DS Homebrew world and further expand my knowledge on things like graphics, FFI, memory representations, etc., and I'm really grateful for that!

I may not be able right now to contribute to very low-level things because I don't know much about the 3DS architecture/ll-details, but I'm interested in learning more about it, and try to help (even if just a little)!
:)

**Comment by Meziu** [comment]
I’ve always appreciated all the help you gave to this project! If you are set on continuing to work on this project like you did up until this day, I’m positive to have you join us “officially”. 

I’d like to know @AzureMarker and @ian-h-chamberlain’s opinion on adding you to the org members.

**Reply by ian-h-chamberlain** [reply]
Absolutely, I think the more the merrier and it's great that more people are interested enough in helping out the project to want to join!

I was also thinking re: pings — it would be nice to have a team for just the active people to avoid spamming any org members that are no longer working on the project, so I just created @rust3ds/active as a convenient way we could mention anyone who's still participating on the project.

**Reply by AzureMarker** [reply]
Sounds good to me! It's always nice to see someone else interested in the project.

**Reply by Techie-Pi** [reply]
Thank you everyone for the invite! :)
